src/index.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
Importing React is not necessary in React 17 and above using the new JSX transform. You can safely remove this import statement 🧹

Reply:
![Screenshot 2023-04-07 at 10 32 30](https://user-images.githubusercontent.com/54940595/230564110-1fcec1a7-00ac-4286-8bca-1cb6be22c3fb.png)